charts-external/openpension/templates/deployment-client.yaml

```diff
@@ -0,0 +1,34 @@
+{{ if .Values.enabled }}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: client
+spec:
+  replicas: 1
+  revisionHistoryLimit: 2
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: client
+    spec:
+      terminationGracePeriodSeconds: 1
+      containers:
+      - name: client
+        image: {{ .Values.client.image }}
+        resources: {{ .Values.client.resources }}
+        ports:
+        - containerPort: 80
+        env:
+        - name: POSTGRES_PASSWORD
+          valueFrom: {"secretKeyRef":{"name":"db", "key":"POSTGRES_PASSWORD"}}
+        - name: DATABASE_URL
+          value: "postgresql://openpension:$(POSTGRES_PASSWORD)@db/openpension"
+        - name: GUNICORN_CMD_ARGS
```

Comment:
I don't think you need these env vars, they are specific to anyway 

also the INITIALIZE env var is probably not relevant